vector/src/main/java/im/vector/activity/MediaPreviewerActivity.java

```diff
@@ -93,8 +100,6 @@ public void onClick() {
         finish();
     }
 
-    //region MediaPreviewAdapter.EventListener
```

Comment:
Why do you remove the `region` marker? @ganfra likes them, it's useful in Android Studio. Please revert